book/src/faq.md

```diff
@@ -88,6 +88,28 @@ involve converting the methods in question to take `&self` argument instead of `
 
 Please consult [`MessageStream`] documentation for details.
 
+## Why aren't property values updating for my service that doesn't notify changes?
+
+A common issue might arise when using a zbus proxy is that your proxy's property values aren't 
+updating. This is due to zbus' default caching policy, which updates the value of a property only
+when a change is signaled, primarily to minimize latency and optimize client request performance.
+By default, if your service does not emit change notifications, the property values will not
+update accordingly. 
+
+However, you can disabling caching for specific properties:
+
+- Add the `#[dbus_proxy(property(emits_changed_signal = "false"))]` annotation to the property
+for which you desire to disable caching on.
+
+- Use `ProxyBuilder` to build your proxy instance and use `ProxyBuilder::uncached_properties` method
+to list all properties you wish to disable caching for.
+
+- In order to disable caching for either type of proxy use the `ProxyBuilder::cache_properites` 
+method.
```

Comment:
`ProxyBuilder::uncached_properties` and `ProxyBuilder::cache_properites` should be links to the docs on these methods.

Reply:
Yup I followed the new convention of adding doc URLS